### PR TITLE
minicom: update to 2.9

### PR DESCRIPTION
**Pull request by setarcos**
#### Testing the changes
- I tested the changes in this PR: briefly

#### Local build testing
- I built this PR locally for my native architecture, x86_64.

### CODE REVIEW

File: srcpkgs/minicom/patches/musl-vc_music.patch
Comment (classabbyamp) [inline srcpkgs/minicom/patches/musl-vc_music.patch:1]: this patch might not be needed, as `__linux__` is probably defined on musl

Comment (setarcos) [reply srcpkgs/minicom/patches/musl-vc_music.patch:1]: Thanks, you are right.
I've updated my PR.

**Comment by setarcos** [comment]
I just realized that the -fcommon CFLAG is no longer necessary. Since this PR has already been merged, it can remain as it is for a little longer.

**Comment by classabbyamp** [comment]
alas

**Pull request merged**